crates/sncast/tests/data/scripts/script_manual_tests/src/test_declare_after_invoke.cairo

```diff
@@ -0,0 +1,45 @@
+use sncast_std::{
+    declare, deploy, invoke, call, DeclareResult, DeployResult, InvokeResult, CallResult, get_nonce,
+};
+use debug::PrintTrait;
+use starknet::{ContractAddress};
+fn main() {
+    let max_fee = 99999999999999999;
+
+    let contract_address: ContractAddress =
+        0x18abd0648d901555975edde1330cffcf37c1da1edbf20b889ba1101e70a6c22
+        .try_into()
+        .expect('Invalid class hash value');
+
+    let invoke_result = invoke(
+        contract_address, 'put', array![0x11, 0x2], Option::Some(max_fee), Option::None
+    );
+
+    let invoke_nonce = get_nonce('pending');
+    invoke_nonce.print();
```

Comment:
I get the intent, though if we were to run those also on devnet I'd remove unnecessary prints in favor of some descriptive comments maybe? For someone who wants to run those on integration/testnet good comment will be a better entry to understand what is happening and what is expected. wdyt?